tests/functional_test.go

```diff
@@ -325,7 +325,7 @@ func TestFunctionalGRPC(t *testing.T) {
 
 	t.Run("TestCheck", func(t *testing.T) { GRPCCheckTest(t, client) })
 	t.Run("TestListObjects", func(t *testing.T) { GRPCListObjectsTest(t, client) })
-
+	t.Run("TestListUsersValidation", func(t *testing.T) { GRPCListUsersTest(t, client) })
```

Comment:
```suggestion
	t.Run("TestListUsers", func(t *testing.T) { GRPCListUsersTest(t, client) })
```
The subtest(s) contained therein would differentiate between validation vs other (non-validation related)

Reply:
Or, rename `GRPCListUsersTest` to `GRPCListUsersValidationTest`